internal/provider/resource_access_generic.go

```diff
@@ -84,11 +83,6 @@ type genericJAASAccessModel struct {
 func (r *genericJAASAccessResource) ConfigValidators(ctx context.Context) []resource.ConfigValidator {
 	return []resource.ConfigValidator{
 		NewRequiresJAASValidator(r.client),
-		resourcevalidator.AtLeastOneOf(
-			path.MatchRoot("users"),
-			path.MatchRoot("groups"),
-			path.MatchRoot("service_accounts"),
-		),
```

Comment:
question:

> Remove AtLeastOneOf validator to allow users to remove all direct access to a resource.

Why leave a resource with no content essentially? Wouldn't the resource just be deleted from the plan? Essentially, why would someone bother?

Reply:
Yes good point, I had a silly moment thinking it would be good to give the user the ability to remove all access but as you said, that's the same as deleting the resource. I've reverted this and fixed the test.